evmchains/__init__.py

```diff
@@ -7,16 +7,19 @@
 import os
 import random
 import re
-from typing import List
+from typing import TYPE_CHECKING, List
 
-from evmchains.chains import PUBLIC_CHAIN_META
-from evmchains.types import Chain
+if TYPE_CHECKING:
+    from evmchains.types import Chain
 
 ENV_VAR_REGEX = re.compile(r"\$\{([A-Za-z0-9_]+)\}")
```

Comment:
I wonder if we should hold off on compiling the regex as well? Potentially doing

```
ENV_VAR_REGEX = r"\$\{([A-Za-z0-9_]+)\}"
```

would suffice